project-base/storefront/components/Forms/validationRules.ts

```diff
@@ -148,9 +148,9 @@ export const validateCompanyTaxNumber = (t: Translate): BaseSchema => {
         );
 };
 
-export const validateFirstPassword = (t: Translate): BaseSchema => {
+export const validatePassword = (t: Translate): BaseSchema => {
     return Yup.string()
-        .required(t('Please enter first password'))
+        .required(t('Please enter password'))
         .min(
```

Comment:
I know this code has been here since ages, but I am thinking if the "repeated" ("again") password should not only be validated by comparison with the "first" password. I think it should be its only validation rule, but this is a question for the PO